src/priya/A19_String.java

```diff
@@ -0,0 +1,32 @@
+package priya;
+
+public class A19_String {
+	
+	int uppercasecnt,lowercasecnt,totalcnt,digitcnt;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.